Content/Items/Gear/Weapons/Melee/Sword.cs

```diff
@@ -0,0 +1,99 @@
+﻿using FunnyExperience.Core.Systems;
+using Terraria.DataStructures;
+using Terraria.ID;
+
+namespace FunnyExperience.Content.Items.Gear.Weapons.Melee
+{
+	internal class Sword : Gear
+	{
+		public override string Texture => $"{FunnyExperience.ModName}/Assets/Items/Gear/Weapon/Sword/Base";
+
+		public override void SetDefaults()
+		{
+			Item.damage = 10;
+			Item.width = 40;
+			Item.height = 40;
+			Item.useStyle = ItemUseStyleID.Swing;
+			Item.useTime = 20;
+			Item.useAnimation = 20;
+			Item.autoReuse = true;
+			Item.DamageType = DamageClass.Melee;
+			Item.knockBack = 6;
+			Item.crit = 6;
+			Item.UseSound = SoundID.Item1;
+			Item.shoot = ProjectileID.PurificationPowder;
+			Item.shootSpeed = 10f;
+			GearType = GearType.Sword;
+		}
+
+		public override bool AltFunctionUse(Player player)
```

Comment:
In the future it's likely a good idea that we simply utilize the base weapon type to track inputs and handle weapon m1/m2 skills in the same skill system as other skills, for consistency and if we want to enable players to augment them later

Reply:
I assumed each base weapon classes, in this case "Sword", would have different alt functions.
While potentially not all weapon classes would have a m2 function.

Reply:
Likely, but I would rather unify both the m1 and m2 functions under the same system as skills on other hotkeys for consistency. Its also unlikely a weapon class would lack an m2 function as that would place it far behind others I would imagine by losing a skill slot.